src/main/java/com/google/sps/servlets/MenteeSignupServlet.java

```diff
@@ -32,6 +33,45 @@
 @WebServlet("/mentee-signup")
 public class MenteeSignupServlet extends HttpServlet {
 
+  /**
+   * Gets majors from database and returns as JSON.
+   */
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Set up map to store majors.
```

Comment:
Rather than explaining exactly what this is doing ("defining a map called majors"), it would be more useful from a readability standpoint to actually explain the "how/why": Why are you using a map? How do the int and string represent a major?

Reply:
Done!